blocks/carousel/carousel.css

```diff
@@ -0,0 +1,488 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+.explore-section {
+    margin-top: 15px;
+    box-sizing: border-box;
+    outline: none;
+}
+
+.explore-section .ideas-strike {
+    background: #FAE4D1;
+    font-weight: 600;
+    padding: 7px;
+    width: fit-content;
+    display: flex;
+    margin-left: auto;
+    margin-bottom: 15px;
+}
+
+.explore-section .ideas-strike img {
+    margin: 0 10px;
+    width: 20px;
+}
+
+.explore-section .explore-container {
+    background-color: #FCFCFC;
+    border: 1px solid rgb(14 31 53 / 20%);
+    border-bottom-left-radius: 15px;
+    border-bottom-right-radius: 15px;
+}
+
+.explore-section .explore-header {
+    background-color: #053d6e;
+    padding: 10px 15px;
+}
+
+@media (width >= 768px) {
```

Comment:
Not sure if this project needs to follow mobile first, if so set `min-width` would be great.

Reply:
@JiangLong2019 My assumption was also to use `min-width` params as well  but linting rules for the same are changed as confirmed by the boiler plate code changes here 
https://github.com/adobe/aem-boilerplate/commit/2f5531c2eabbe05fc95681bce52279bc3089fff3
hence operator base calculation.